src/main/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/service/MessageNotifier.kt

```diff
@@ -17,10 +23,18 @@ private const val MESSAGE_GROUP_ID = "COURT_HEARING_EVENT_RECEIVER"
 class MessageNotifier(
   private val objectMapper: ObjectMapper,
   private val hmppsQueueService: HmppsQueueService,
+  @Autowired private val amazonS3AsyncClient: S3AsyncClient,
```

Comment:
The @Autowired annotation here isn't doing anything - @Component means that the constructor arguments will be autowired anyway